src/i18n/config.ts

```diff
@@ -21,3 +21,5 @@ i18next.use(initReactI18next).init({
     // set returnNull to false (and also in the i18next.d.ts options)
     // returnNull: false,
 });
+
+export default i18next;
```

Comment:
There's a lot of good things you're noticing, though they are out of scope.

Better to have one PR for one task, and keep 'em small 🤏🏻